frontend/src/model/Client.ts

```diff
@@ -0,0 +1,4 @@
+export type Client = {
+    id: string,
```

Comment:
You can make id optional, so you are not forced to set an id for the axios request ;-)

```suggestion
    id?: string,
```